sdk/nodejs/policy/branchPolicyBuildValidation.ts

```diff
@@ -128,11 +128,11 @@ export class BranchPolicyBuildValidation extends pulumi.CustomResource {
      * @param args The arguments to use to populate this resource's properties.
      * @param opts A bag of options that control this resource's behavior.
      */
-    /** @deprecated azuredevops.policy.BranchPolicyBuildValidation has been deprecated in favor of azuredevops.BranchPolicyBuildValidation */
+    /** @deprecated azuredevops.policy/branchpolicybuildvalidation.BranchPolicyBuildValidation has been deprecated in favor of azuredevops.index/branchpolicybuildvalidation.BranchPolicyBuildValidation */
```

Comment:
This doesn't look right @iwahbe

Reply:
This is how our auto-aliasing code inserts deprecation messages. It's a slight regression in typescript, but slightly clearer in other languages.